cmd/otelcorecol/builder-config.yaml

```diff
@@ -2,29 +2,29 @@ dist:
   module: go.opentelemetry.io/collector/cmd/otelcorecol
   name: otelcorecol
   description: Local OpenTelemetry Collector binary, testing only.
-  version: 0.43.1-dev
```

Comment:
What does this version affect? Did we incorrectly release 0.44.0 without updating this field?

Reply:
No, this is what's displayed in the logs when you run `make run`. This distribution is only used for local debugging. The other versions on this file also don't matter that much, as the replaces statement down there points to the file system location for the modules.